src/lib/get-mediatype-examples.js

```diff
@@ -1,92 +1,77 @@
 const { sampleFromSchema } = require('../samples');
-const cleanStringify = require('./json-stringify-clean');
 const matchesMimeType = require('./matches-mimetype');
 
-module.exports = {
-  /**
-   * Extracts an example from an OAS Media Type Object. The example will either come from the `example` property, the
-   * first item in an `examples` array, or if none of those are present it will generate an example based off its
-   * schema.
-   *
-   * @link https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#mediaTypeObject
-   * @param {string} mediaType
-   * @param {object} mediaTypeObject
-   * @param {object} opts Configuration for controlling `includeReadOnly` and `includeWriteOnly`.
-   * @returns {(object|false)}
-   */
-  getMediaTypeExample: (mediaType, mediaTypeObject, opts = {}) => {
-    if (mediaTypeObject.example) {
-      return mediaTypeObject.example;
-    } else if (mediaTypeObject.examples) {
-      const examples = Object.keys(mediaTypeObject.examples);
-      if (examples.length) {
-        if (examples.length > 1) {
-          // Since we're trying to return a single example with this method, but have multiple present,
-          // return `false` so `getMultipleExamples` will pick up this response instead later.
-          return false;
-        }
+/**
+ * Extracts an array of examples from an OpenAPI Media Type Object. The example will either come from the `example`
+ * property, the first item in an `examples` array, or if none of those are present it will generate an example based
+ * off its schema.
+ *
+ * @link https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#mediaTypeObject
+ * @param {string} mediaType
+ * @param {object} mediaTypeObject
+ * @param {object} opts Configuration for controlling `includeReadOnly` and `includeWriteOnly`.
+ * @returns {array}
+ */
+module.exports = function getMediaTypeExamples(mediaType, mediaTypeObject, opts = {}) {
+  if (mediaTypeObject.example) {
+    return [
+      {
+        value: mediaTypeObject.example,
+      },
+    ];
+  } else if (mediaTypeObject.examples) {
+    const { examples } = mediaTypeObject;
+    const multipleExamples = Object.keys(examples)
+      .map(key => {
+        let summary = key;
```

Comment:
I feel like there's value in preserving both the `key` here **and** a hypothetical `example.summary`. What about storing this as a `title` value or something? So like in this example:
```yml
foo:
  summary: A foo example
  value: {"foo": "bar"}
bar:
  summary: A bar example
  value: {"bar": "baz"}
```
we'd set the `multipleExamples` array to something like:
```json
[
  {
    "summary": "A foo example",
    "title": "foo",
    "value": { "foo": "bar" }
  },
  {
    "summary": "A bar example",
    "title": "bar",
    "value": { "bar": "baz" }
  }
]
```
 Based on [the spec](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#example-object), it seems like the `summary` contains descriptive information that seems like it's playing a different role than the key of the object.

Reply:
I like this.

Reply:
Added in https://github.com/readmeio/oas/pull/470/commits/2327feec10347b82a964b15ff4de31bbb5e18a8d